### ISSUE TITLE
enh: Modifying (launch?) environment

### ISSUE THREAD

**Issue by lanodan** [swaywm/sway#6181]
I wanted to modify the environment programs would get launched in and I couldn't find a documented way in sway manpages, I know one solution is to attach a debugger but that's quite a recipe for trouble if you don't do it carefully.

I think there should be commands like `env_unset` and `env_set` (or `env_export` to be similar to a unix shell) in sway for this.

**Comment by Emantor** [comment]
Simply setup the environment variables before starting sway on the command line.

**Comment by emersion** [comment]
Duplicate of https://github.com/swaywm/sway/issues/3976
